test/mlir/conversion/onnx_to_torch/leaky_relu/leakyalpha_one.onnx.mlir

```diff
@@ -0,0 +1,10 @@
+//RUN: onnx-mlir --EmitONNXIR --run-torch-pass %s -o=%t >/dev/null && cat %t.onnx.mlir | FileCheck -v %s
+module attributes {llvm.data_layout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128", llvm.target_triple = "x86_64-unknown-linux-gnu"}  {
+  func @main_graph(%arg0: tensor<5xf32>) -> tensor<5xf32> attributes {input_names = ["input"], output_names = ["1"]} {
+//CHECK: [[ALPHA:%[^ ]*]] = torch.constant.float -1.000000e+01
+//CHECK: torch.aten.leaky_relu %arg0, [[ALPHA]] :
+    %0 = "onnx.LeakyRelu"(%arg0) {alpha = -1.000000e+01 : f32, onnx_node_name = "LeakyRelu_0"} : (tensor<5xf32>) -> tensor<5xf32>
```

Comment:
This seems to be the same test as `test/mlir/conversion/onnx_to_torch/leaky_relu/leaky_relu.onnx.mlir`. If so, please delete one of them.

Reply:
Hi Liam,
We have removed one test case which was similar. Now there are two, one written by yourself and the other from our side. The CHECK statements seem to be the same although the attributes are different. Let us know if you want us to remove the other one also.

Reply:
Please delete the one I had added - it has a bunch of non-standard attributes that add nothing to the test. Yours is cleaner.

Reply:
Hi Liam,
We have removed the test case.